website/source/guides/operations/reference-architecture.html.md

```diff
@@ -0,0 +1,133 @@
+- [Deployment System Requirements](#system-reqs)
+- [High Availability](#high-availability)
+- [Failure Scenarios](#failure-scenarios)
+
+This document describes deploying a Nomad cluster in combination with, or with access to, a [Consul cluster](/guides/operations/consul-integration/index.html). We recommend the use of Consul with Nomad to provide automatic clustering, service discovery, health checking and dynamic configuration.
+
+## <a name="ra"></a>Reference Architecture
+
+A Nomad cluster typically comprises three or five servers (but no more than seven) and a number of client agents. A cluster can be deployed in a single physical datacenter or it can span multiple datacenters/regions. There is an important distinction to make in that Nomad divides infrastructure into regions, which are served by one Nomad server cluster, but can manage multiple datacenters. For example, a _US Region_ can include datacenters _us-east-1_ and _us-west-2_.
+
+In a Nomad multi-datacenter/multi-region architecture, communication happens via [WAN gossip](/docs/internals/gossip.html). Additionally, Nomad can integrate easily with Consul to provide features such as automatic clustering, service discovery, and dynamic configurations. Thus we recommend you use Consul in your Nomad deployment to simplify the deployment.
+
+In cloud environments, a single datacenter may be deployed across multiple availability zones. For example, in AWS each Nomad server can be deployed to an associated EC2 instance, and those EC2 instances distributed across multiple AZs. Similarly, Nomad server clusters can be deployed to multiple cloud regions to allow for region level HA scenarios.
+
+For more information on Nomad server cluster design, see the [cluster requirements documentation](/guides/operations/requirements.html).
+
+The design shared in this document is the recommended architecture for production environments, as it provides flexibility and resilience. Nomad utilizes an existing Consul server cluster; however, the deployment design of the Consul server cluster is outside the scope of this document.
+
+Nomad to Consul connectivity is over HTTP and should be secured with TLS as well as a Consul token to provide encryption of all traffic. This is done using Nomad's [Automatic Clustering with Consul](/guides/operations/cluster/automatic.html).
+
+### <a name="one-dc"></a>Deployment Topology within One Datacenter
+
+A single Nomad cluster is recommended for applications deployed in the same region.
+
+Each cluster is expected to have either three or five servers. This strikes a balance between availability in the case of failure and performance, as [Raft](https://raft.github.io/) consensus gets progressively slower as more servers are added.
+
+The time taken for gossip to converge increases as more client nodes are added. Similarly, the time taken by a new server to join an existing large cluster may increase as they are replicated to the new server's log.
```

Comment:
suggested replace `Similarly, the time taken by a new server to join an existing large cluster may increase as they are replicated to the new server's log.`  with `A new server that joins an existing cluster takes longer to replicate data as the size of the cluster increases`